scripts/bin/GenerateStaticData.pm

```diff
@@ -704,6 +704,7 @@ sub _texts {
         push @texts, localize('Adjusted High Barrier');
         push @texts, localize('Adjusted Low Barrier');
         push @texts, localize('Corporate Action');
+        push @texts, localize('The final settlement price may differ from the Indicative price.');
 
```

Comment:
@yngshan this is removed in my latest commit... git keeps the message history of old code in this page, please ignore it